### PR TITLE
Scratching my own itches

### PR DESCRIPTION
**Pull request by sixtyfive**
This is just the result of me getting on people's nerves on IRC and being told to open a PR here.

### CODE REVIEW

File: packaging/package.yml/en.md [hunk 1]
```diff
@@ -18,12 +18,15 @@ An example file follows:
...
 name       : nano
 version    : 2.5.1
+# Release has nothing to do with upstream. It's just to know whether the package manager should upgrade so we can downgrade a version and increment release so that the package manager downgrades users if there are issues with a particular version.
 release    : 39
 source     :
     - http://www.nano-editor.org/dist/v2.5/nano-2.5.1.tar.gz : e06fca01bf183f4d531aa65a28dffc0e2d10185239909eb3de797023f3453bde
 license    : GPL-3.0
 component  : editor
 summary    : Small, friendly text editor inspired by Pico
+builddeps  :
```
Comment (ikeydoherty) [inline packaging/package.yml/en.md]: This kinda makes a trivial example non-trivial, but I'm on the fence about it. At least folks will know from the get go how to builddep..

Comment (sixtyfive) [inline packaging/package.yml/en.md]: What if I made that line a comment, too? Would that get you off the fence? :)

Comment (ikeydoherty) [inline packaging/package.yml/en.md]: IDK maybe we just keep it in. The problem is multiple..
`ncurses-devel` is in `system.devel` so its implicitly available (redundant builddep)
Plus it provides pkgconfig names which should be used instead like `pkgconfig(ncursesw)`

**Comment by JoshStrobl** [comment]
Gonna be honest, not really sure why this PR is needed. It's already made clear in the Package.yml table what version is for, there's no reason for it to be in the example, nor any reason to add an unnecessary builddep in the builddeps section, when we have a dedicated sections on learning about build and rundeps, both of which are linked on the Package.yml page (in the table), to Packaging Practices.

**Pull request closed without merging**